### ISSUE TITLE
about clip model

### ISSUE THREAD

**Issue by zhangxh2414** [xBimTeam/XbimWindowsUI#124]
I find in the WebUI there have a function that have method to clip the model.
Can WindowsUI do that as well？

**Comment by andyward** [comment]
Yes, it does. Try holding down 'Shift' and clicking. You get an axis object that you can drag around and re-orientate to clip the model in different orientations / levels.

![image](https://user-images.githubusercontent.com/923699/62796614-8ee7d500-bad1-11e9-907b-c0383b08fe68.png)

It's slightly fiddly (you have to click the right part of the axis in 3D to move the cutplane up/around) but you should be able to see the concept.
